### PR TITLE
Add files for test deps for Fedora 32

### PR DESCRIPTION
**Pull request by adityapatwardhan**
## PR Summary

Add test deps image for stable and preview for Fedora 32

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/stable/fedora32/test-deps/meta.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "IsLinux": true,
+    "UseLinuxVersion": false,
+    "osVersion": "Fedora 32",
+    "tagTemplates": [
+        "preview-fedora-#shorttag#"
```
Comment (TravisEz13) [inline release/stable/fedora32/test-deps/meta.json:6]:
```suggestion
        "fedora-#shorttag#"
```

Comment (adityapatwardhan) [reply release/stable/fedora32/test-deps/meta.json:6]: yes, that explains the dup

File: release/preview/fedora32/test-deps/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Docker image file that describes an Fedora32 image with PowerShell installed from Microsoft YUM Repo
+ARG BaseImage=mcr.microsoft.com/powershell:fedora-32
```
Comment (TravisEz13) [inline release/preview/fedora32/test-deps/docker/Dockerfile:2]:
```suggestion
ARG BaseImage=mcr.microsoft.com/powershell:preview-fedora-32
```

**Comment by adityapatwardhan** [comment]
@TravisEz13 please review again

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**